gbbs/benchmarks/BatchDynamicConnectivity/SkipList/SkipList.h

```diff
@@ -540,18 +548,14 @@ inline void RunSkipList(uintE n) {
     skip_list.batch_join(&join_updates);
 
     std::cout << "printing answers" << std::endl;
+    
+    for (int i = 0; i < 6; ++i) {
+    std::cout << "node " << i + 1 << " height: " << skip_list_elements[i].height << std::endl;}
```

Comment:
```suggestion
    
    for (int i = 0; i < 6; ++i) {
        std::cout << "node " << i + 1 << " height: " << skip_list_elements[i].height << std::endl;
    }
```
Good! :smile: Just pay close attention to indentation and whitespace.